plugin-server/src/worker/vm/vm.ts

```diff
@@ -58,8 +78,6 @@ export function createPluginConfigVM(
 
     // Add non-PostHog utilities to virtual machine
     vm.freeze(AVAILABLE_IMPORTS['node-fetch'], 'fetch')
-    vm.freeze(createGoogle(), 'google')
-
     vm.freeze(AVAILABLE_IMPORTS, '__pluginHostImports')
```

Comment:
This is where we stuff _all_ of our import code into the VM. I think we should use the new `usedImports` above and only inject code for ones that are used, but I don't want to ship that quickly on a Friday.